canonical-web-and-design/snapcraft.io
# Email preferences

Show's whether a publisher is signed up to the blog newsletter and allows them to toggle their subscription.

## QA

- You'll need `MARKETO_URL`, `MARKETO_CLIENT_ID` and `MARKET_CLIENT_SECRET` in your `.env.local` (if you're familiar with Marketo, go ahead and get them - otherwise ask me :))
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account/details
- You should see an "Email preferences" section at the bottom of the page with a toggle for subscribing
- Toggle the checkbox and save
- Ensure toggling/ untoggling and saving works as you'd expect
